docs/installation/instructions.rst

```diff
@@ -77,15 +76,13 @@ PyPI Installation
    $ pulp-manager migrate --noinput
    $ pulp-manager reset-admin-password --password admin
 
-9. Collect Static Media for live docs and browsable API
+9. Collect Static Media for live docs and browsable API::
 
    $ pulp-manager collectstatic --noinput
 
-10. Run Pulp:
-::
-
-   $ django-admin runserver
+10. Run Pulp::
 
+    $ pulp-manager runserver
```

Comment:
Changed to `pulp-manager` for consistency here (since it's used in all the other instructions above).